CHANGELOG.md

```diff
@@ -1,19 +1,18 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* [Sync] Emit a `startup` event to the Sync event listener when a Realm matching the regex is detected upon the listener startup.
+* Emit a `startup` event to the `Realm.Sync` event listener when a Realm matching the regex is detected upon the listener startup. ([realm-js-private#521](https://github.com/realm/realm-js-private/issues/521))
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* Fixes an issue where calling `user.logout()` would not revoke the refresh token on the server. (Since v2.24.0)
+* Fixes an issue where calling `user.logout()` would not revoke the refresh token on the server. ([#2348](https://github.com/realm/realm-js/pull/2348), since v2.24.0)
 
 ### Compatibility
 * Realm Object Server: 3.11.0 or later.
 * APIs are backwards compatible with all previous release of realm in the 2.x.y series.
 * File format: Generates Realms with format v9 (Reads and upgrades all previous formats)
 
 ### Internal
-* None.
+* Update dependencies and fix vulnerabilities.
```

Comment:
This is meaningless IMO, as a minimum we should write which dependencies.

Reply:
agree. Was there any relevant vulnerability that affects any users?

Reply:
Maybe "Building using Xcode 10 supported" is better.